plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -160,6 +161,30 @@ const applyConditions = <TResourceType extends string, TResource>(
   return rule.apply(resource, criteria.params ?? {});
 };
 
+/**
+
+ * Takes some permission conditions and returns a definitive authorization result
+ * on the resource to which they apply.
+ *
+ * @public
+ */
+export const createIsAuthorized = <TResource, TQuery>(
```

Comment:
Do you think it would make sense to more clearly pair this up with `createConditionTransformer`? From my understanding, this is the immediate version where we have the resource, while the condition transformer turn the conditions into the resource-specific query. I'm thinking something like `createConditionApplicator`?

Reply:
`createAuthorizer`?
I honestly would prefer to expose `createGetRule` method and keep `createIsAuthorized` as scaffolder internal. I feel it's hard to explain what `createIsAuthorized` is supposed to do

Reply:
If we export `createGetRule` we'd also need to export `applyConditions` right? Feels like those two are a bit lower level and it makes sense to bundle them together into a single utility tbh.

This is all about conditions still though right? So I'm feeling that keeping the name of the form `createCondition*` would be desirable, especially with the existence of `createConditionTransformer`.

Reply:
I buy `createConditionAuthorizer` 💰